src/OneTrustCookieConsentBanner.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC, useEffect } from 'react';
+
+type Props = {
+  domainScript: string;
+  enabled: boolean;
+};
+
+const OneTrustCookieConsentBanner: FC<Props> = ({ domainScript, enabled }) => {
+  const src = 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js';
+
+  const loadOneTrust = () => {
+    const preloadLink = document.createElement('link');
```

Comment:
I don't think this will have an effect if you add it client side in the useEffect, am I missing something?

Reply:
as discussed, removed it completely because it has no effect when both tags are placed in the head